networkx/algorithms/independence.py

```diff
@@ -0,0 +1,52 @@
+import networkx as nx
+from networkx.algorithms import *
```

Comment:
import the functions you are specifically using i.e. `enumerate_all_cliques` and `complement`